### ISSUE TITLE
Heading block uses a counter-intuitive icon

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15340]
Heading block uses a counter-intuitive icon

-   **Severity:** Low
-   **Affected Populations:**
    -   Low-Vision
    -   Cognitively Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -   Block Editing

#### Issue description

The "Block Type" menu, and block editor type button, use icons for
each block, and in most cases these are good representations of the
block. However the "Heading" block type uses the letter "T", which
is potentially confusing, since "Heading" doesn't start with "T".

Users with cognitive disabilities, especially those who are more
icon-reliant, may mistake the heading type for something else (such as
"Table" or "Text"), especially in cases where only the icon is
visible (such as in the block editor button, or when the "Block Type"
menu list is scrolled so that only the icon is visible).

#### Remediation Guidance

Use the letter "H" as the icon for the Heading block type. This also
matches the other H icons used for setting the level (H2, H3, etc.).

#### Relevant standards

-   [1.3.3 Sensory Characteristics
    (Level A)](https://www.w3.org/TR/WCAG20/#content-structure-separation-understanding)

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-102 in Tenon's report

**Comment by mapk** [comment]
I'm not sure about the decision behind using the "T" but it could cognitively relate to "Title" as in the case of [Material Icons](https://material.io/tools/icons/?icon=title&style=baseline).

<img width="123" alt="Screen Shot 2019-05-01 at 9 41 56 AM" src="https://user-images.githubusercontent.com/617986/57028994-69574f80-6bf5-11e9-97e5-f38c47fce010.png">


Also when looking at the case of [formatting size](https://material.io/tools/icons/?icon=format_size&style=baseline), Material uses a small and larger "T" as the icon.  

<img width="122" alt="Screen Shot 2019-05-01 at 9 42 04 AM" src="https://user-images.githubusercontent.com/617986/57029001-6d836d00-6bf5-11e9-9c02-5aff2782efb9.png">


I'm not opposed to using an "H" but would like to hear other design feedback first.

**Comment by sarahmonster** [comment]
I believe there was a discussion about this some time ago, but I can't find it. The crux was that H wasn't inclusive of non-English languages. 

We do use H for the heading levels, which feels a bit inconsistent:

<img width="202" alt="Screenshot 2019-05-01 at 10 49 36" src="https://user-images.githubusercontent.com/376315/57032317-ed620500-6bfe-11e9-9437-b1240e517176.png">

From a user experience perspective, my guess is that `H` is a more universal symbol for a heading element. That said, that might be coloured by my own experience as familiar with HTML and English-native. It would be super helpful to get some feedback from a variety of different languages here—maybe we could try a few different ideas and see what people respond best to.

For reference, Dropbox Paper uses "H1" and "H2", as does Bear. I haven't found many other instances of heading icons, but it's probably worth looking to see what standards may be established elsewhere.

**Comment by brentswisher** [comment]
@sarahmonster Possible #9313 is what you were thinking of?

**Comment by mapk** [comment]
In [today's slack triage](https://wordpress.slack.com/archives/C02S78ZAL/p1556820639448800), there was consensus around switching the "T" icon to an "H" because it related to the "H1", "H2", etc. tags in markup and might be more easily understood across languages.

Our next task is to get a PR created around this.
